Cargo.toml

```
@@ -36,11 +36,11 @@ anyhow = { version = "1.0.1", optional = true }
 chrono = { version = "0.4.25", default-features = false, optional = true }
 chrono-tz = { version = ">= 0.6, < 0.11", default-features = false, optional = true }
```

Ah, looks like this got missed:

```suggestion
chrono-tz = { version = ">= 0.10, < 0.11", default-features = false, optional = true }
```

... I'm not sure why we have it in `dev-dependencies` as well 🤔

---

Updated.